packages/front-end/components/Features/SDKEndpoints.tsx

```diff
@@ -93,71 +92,58 @@ const SDKEndpoints: FC<{
                   </td>
                   <td>
-                    <code>{endpoint}</code>{" "}
-                    <FaCopy
-                      className="cursor-pointer"
-                      title="Copy to Clipboard"
-                      onClick={(e) => {
-                        e.preventDefault();
-                        navigator.clipboard
-                          .writeText(endpoint)
-                          .then(() => {
-                            console.log("Copied!");
-                          })
-                          .catch((e) => {
-                            console.error(e);
-                          });
-                      }}
-                    />
+                    <ClickToCopy valueToCopy={endpoint}>
+                      <span style={{ overflowWrap: "anywhere" }}>
+                        {endpoint}
+                      </span>
+                    </ClickToCopy>
                   </td>
                   {hasEncryptedEndpoints && (
-                    <td>
-                      {key.encryptSDK ? (
-                        canManageKeys ? (
-                          <ClickToReveal
-                            valueWhenHidden="Reveal key"
-                            getValue={async () => {
-                              const res = await apiCall<{
-                                key: ApiKeyInterface;
-                              }>(`/keys/reveal`, {
+                    <td style={{ width: 295 }}>
+                      {canManageKeys && key.encryptSDK ? (
+                        <ClickToReveal
+                          valueWhenHidden="Click to reveal encryption key"
+                          getValue={async () => {
+                            const res = await apiCall<{ key: SecretApiKey }>(
+                              `/keys/reveal`,
+                              {
                                 method: "POST",
                                 body: JSON.stringify({
                                   id: key.id,
                                 }),
-                              });
-                              if (!res.key?.encryptionKey) {
-                                throw new Error(
-                                  "Could not load encryption key"
-                                );
                               }
-                              return res.key.encryptionKey;
-                            }}
-                          >
-                            {(value) => <CopyToClipboard text={value} />}
-                          </ClickToReveal>
-                        ) : (
-                          "yes"
-                        )
+                            );
+                            if (!res.key.key) {
+                              throw new Error(
+                                "Could not load the encryption key"
+                              );
+                            }
+                            return res.key.key;
```

Comment:
This is wrong.  It should be returning `encryptionKey`.  Right now it's just returning the SDK endpoint key.  I would revert this whole `getValue` function since I think it was correct before.